# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/malloc-defined.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- isdn4k-utils/isdnlog/tools/cdb/cdbmake.c~	2002-02-22 12:10:25.000000000 +0100
+++ isdn4k-utils/isdnlog/tools/cdb/cdbmake.c	2004-10-14 13:53:26.383237096 +0200
@@ -3,7 +3,7 @@
 #include "freecdbmake.h"
 #include <unistd.h>
 
-extern char *malloc();
+/*extern char *malloc();*/
 
 void diesys(why) char *why; { perror(why); exit(111); }
 
